linode/manager
# M3-5101: Storybook beta stories

## Description 📝

**What does this PR do?**
Documents the beta components we use, to improve ease of rolling out future beta features:

- Adds a subpage for the Beta chip on the Storybook [Chips page](https://design.linode.com/?path=/docs/elements-chips--default-story)
   - In order to avoid reconstructing more complex components as stories just to demo with a beta chip, the docs include a link to a previous PR where a screenshot and code can be referenced.
- Adds a subsection for the Beta banner on the Storybook [Dismissible Banners page](https://design.linode.com/?path=/docs/components-notifications-dismissible-banners--dismissible-banners)
- ~~Updates a Chip to create a 'beta' variant to improve reusability~~ Creates a new BetaChip component
- Updates existing Chips with beta styling to the ~~beta variant~~ BetaChip


## How to test 🧪

**What are the steps to reproduce the issue or verify the changes?**
- Run `yarn storybook` to start a local instance of Storybook. Verify that the beta sections on the Beta Chip and Dismissible Banner stories display, the links work correctly (beta chips and beta banners should link to each other's stories), and documentation seems thorough.
- Toggle the database beta feature flag with the MSW and check that beta chip variant styling appears consistent with beta chip styling prior to adding variant.
  - Will want to navigate to the Database Create page to see both the primary nav and breadcrumb chips.
  - Toggle dark mode and verify chip turns blue.
